pylint_odoo/test/main.py

```diff
@@ -61,6 +61,7 @@
     'except-pass': 3,
     'attribute-string-redundant': 33,
     'renamed-field-parameter': 2,
+    'deprecated-data-xml-node': 4,
```

Comment:
Consider add a xml with the correct case without data node!
`<odoo>`
`<odoo noupdate="1">`
`<odoo noupdate="0">`